sync/src/main/kotlin/gropius/sync/AbstractSync.kt

```diff
@@ -282,16 +290,29 @@ abstract class AbstractSync(
             issue = dereplicationResult.resultingIssue
             for (fakeSyncedItem in dereplicationResult.fakeSyncedItems) {
                 nodesToSave.add(fakeSyncedItem)
-                savedNodeHandlers.add {
-                    val tici = DummyTimelineItemConversionInformation(imsProject.rawId!!, (it as TimelineItem).rawId!!)
-                    tici.gropiusId = (it as TimelineItem).rawId
+                savedNodeHandlers.add { updatedNode ->
+                    val tici = DummyTimelineItemConversionInformation(
```

Comment:
Better variable name?
Also, why "dummy"?

Reply:
no. changing this into`timelineItemConversionInformation` would turn the code into unreadable comedy.

Dummy, because it's for fake timelineItems inserted by the dereplicator, which you could have seen yourself

Reply:
0495a0e6fe8a26054ac4908f7a1ce8e0011c8a87